client/Overview/FeaturedReviewLink.jsx

```diff
@@ -1,25 +1,25 @@
-import React from 'react';
-// import Stars from '../SharedComponents/Stars.jsx';
+// import React from 'react';
+// // import Stars from '../SharedComponents/Stars.jsx';
 
-// Kyle: I did also create a Stars component that may be of use to you. It's in the SharedComponents folder. All you have to pass it is a rating number as a prop, and it will give you an appropriate amount of filled stars.
+// // Kyle: I did also create a Stars component that may be of use to you. It's in the SharedComponents folder. All you have to pass it is a rating number as a prop, and it will give you an appropriate amount of filled stars.
 
-// function ScrollToReviewSection() {
-//   document.getElementByClassName('ratings-and-reviews-container').scrollIntoView(true)
-// }
+// // function ScrollToReviewSection() {
+// //   document.getElementByClassName('ratings-and-reviews-container').scrollIntoView(true)
+// // }
 
-function FeaturedReviewLink() {
-// const FeaturedReviewLink = () => {
-// error: Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style
-  return (
-    <div id="productAreaStars">
-      <strong>[*****]</strong>
-      <span> Read all reviews</span>
-    </div>
-    // <div id="productAreaStars">
-    //   <strong>[*****]</strong>
-    //   <span onClick={this.ScrollToReviewSection}>Read all reviews</span>
-    // </div>
-  );
-}
+// function FeaturedReviewLink() {
+// // const FeaturedReviewLink = () => {
+// // error: Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style
```

Comment:
To fix this, try replacing the curly braces on lines 11 and 23 with parentheses, then deleting lines 13 and 22 and aligning the remaining content properly. I believe the linter doesn't like redundant return statements.